src/main/js/main_html5.js

```diff
@@ -284,15 +284,19 @@ require("../../../html5-common/js/utils/environment.js");
                     "play": raisePlayEvent,
                     "pause": raisePauseEvent,
                     "ratechange": raiseRatechangeEvent,
-                    "volumechange": raiseVolumeEvent,
-                    "volumechangeNew": raiseVolumeEvent,
                         // ios webkit browser fullscreen events
                     "webkitbeginfullscreen": raiseFullScreenBegin,
                     "webkitendfullscreen": raiseFullScreenEnd
                   };
       // events not used:
       // suspend, abort, emptied, loadeddata, resize, change, addtrack, removetrack
       _.each(listeners, function(v, i) { $(_video).on(i, v); }, this);
+      // The volumechange event does not seem to fire for mute state changes when using jQuery
+      // to add the event listener or when using addEventListener("volumechange"). It does
+      // work using the below line. We need this event to fire properly or else other SDKs
+      // (such as the Freewheel ad SDK) that make use of this video element may have issues with
+      // the mute state
+      _video.onvolumechange = raiseVolumeEvent;
```

Comment:
Would this work with `addEventListener()`?

Reply:
It didn't from my tests, I mentioned this in the comment.

Reply:
Oh ok, my brain just left out that part of the comment. I thought I had read it all.

Reply:
addEventListener works, will be making use of it.